hi-ml/src/health_ml/utils/lightning_loggers.py

```diff
@@ -109,3 +112,33 @@ def to_metrics_dicts(self, prefix_filter: str = "") -> Dict[int, DictStrFloat]:
         :return: A dictionary mapping from epoch number to metric name to metric value.
         """
         return {epoch: self.extract_by_prefix(epoch, prefix_filter) for epoch in self.epochs}
+
+
+class MlflowLogger(MLFlowLogger):
```

Comment:
I find it confusing to have two classes that only differ by casing.

Reply:
Yes I wasn't sure what to call this...any ideas? HimlMlflowLogger?